contracts.go

```diff
@@ -1,109 +1,109 @@
 package main
 
 import (
-	"bytes"
-	"crypto/sha256"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"path/filepath"
-	"regexp"
-	"strings"
-	"text/template"
-
-	"github.com/spf13/cobra"
+  "bytes"
+  "crypto/sha256"
+  "fmt"
+  "io/ioutil"
+  "os"
+  "os/exec"
+  "path/filepath"
+  "regexp"
+  "strings"
+  "text/template"
+
+  "github.com/spf13/cobra"
```

Comment:
yea ethan noticed it on my other pull request. for some reason atom absolutely wrecks the formatting. i downloaded a go plugin but forgot to go back and change the fmt on this commit. sorry!